projects/aca-playwright-shared/src/page-objects/components/adf-info-drawer.component.ts

```diff
@@ -36,4 +36,35 @@ export class AdfInfoDrawerComponent extends BaseComponent {
   public getIdField = (labelText: string) => this.getChild('[data-automation-id="library-id-properties-wrapper"]', { hasText: labelText });
   public getVisibilityField = (labelText: string) => this.getChild('[data-automation-id="library-visibility-properties-wrapper"]', { hasText: labelText });
   public getDescriptionField = this.getChild('[data-automation-id="library-description-properties-wrapper"] textarea');
+  public propertiesTab = this.getChild('.adf-info-drawer-tab').nth(0);
+  public commentsTab = this.getChild('.adf-info-drawer-tab').nth(1);
+  public commentInputField = this.getChild('mat-form-field');
+  public commentsHeader = this.getChild('#comment-header');
+  public addCommentButton = this.getChild('[data-automation-id="comments-input-add"]');
+  public commentsList = this.getChild('.adf-comment-list-item');
+  public commentUsername = this.getChild('.adf-comment-user-name');
+  public commentTextContent = this.getChild('.adf-comment-message');
+  public commentTimestamp = this.getChild('.adf-comment-message-time');
+
+
+  async checkCommentsHeaderCount(): Promise<number> {
+    const commentsCountTextContent = await this.commentsHeader.textContent();
+    const commentsCountString = commentsCountTextContent.match(/\d+/g)[0];
+    return parseInt(commentsCountString);
+  }
+
+  async verifyCommentsCountFromList(expectedNumber: number): Promise<void> {
+    const commentsCountFromList = await this.commentsList.count();
+    expect(commentsCountFromList).toEqual(expectedNumber);
+  }
+
+  async waitForComments(): Promise<void> {
```

Comment:
Do you even need to wrap it inside a method? It's just a one line

Reply:
I decided to make a method for that, so the code looks more clear, similar as it is in Protractor. I agree that it is not needed, so I can delete that. Please decide if it should be deleted. @MichalKinas

Reply:
Okay in this case let's leave it